src/integration_tests/dynamo.integration.test.ts

```diff
@@ -57,4 +57,27 @@ describe("dynamo integration tests", () => {
       )
     );
   });
+
+  describe("CRUD on character meta table", () => {
+    test("it does the thing", async () => {
```

Comment:
The test name might be funny, but we're not here to be funny, are we Brian? We're here to make SOFTWARE like PROFESSIONAL ADULTS! 
Code Readability: F

Reply:
True that, I only 0.12456 ass'd making these tests.